source/node.go

```diff
@@ -57,7 +57,11 @@ func NewNodeSource(ctx context.Context, kubeClient kubernetes.Interface, annotat
 	nodeInformer.Informer().AddEventHandler(
 		cache.ResourceEventHandlerFuncs{
 			AddFunc: func(obj interface{}) {
-				log.Debug("node added")
+				node, ok := obj.(*v1.Node)
```

Comment:
Could you add a test. There test helper for logs https://github.com/kubernetes-sigs/external-dns/blob/be7f9701222563f7a7513168c2defd962875c332/internal/testutils/log.go#L36